### ISSUE TITLE
Any update?

### ISSUE THREAD

**Issue by greatjack1** [imasaru/android_device_zte_oldman_twrp#3]
I also have this phone, and Im trying to get access to /system and so far have been unsuccessful. Any update?

**Comment by imasaru** [comment]
@greatjack1 Nope... I have tried various applications like KingoRoot, KingRoot, and exploits like dirtyc0w for gaining root access, and nothing has worked. I've since switched to another phone, but if there's root success with this one, I'll gladly update the tree and build for this device :smile:
